FAQ's.md

```diff
@@ -0,0 +1,30 @@
+**Q: What is ADSdroid ?**
+
+A: It is the unofficial AllDataSheet Android Application. It gives her users the privilege of searching and downloading electronics tools in pdf format, for easier and faster access 
+
+**Q: Does it requires internet access ?**
+
+A: Absolutely. It requires internet connection, for searching and also for downloading files in PDF format.
+
+**Q: How do I use it ?**
+
+A: Very simple. Just input a valid electrical terms in the search box and click on the search button 
```

Comment:
`input a valid electrical terms` should be `enter the name of the component (can be partial)`